pyteal/ast/ecdsa.py

```diff
@@ -0,0 +1,153 @@
+from enum import Enum
+from typing import Tuple, TYPE_CHECKING, Union, cast
+
+from pyteal.ast import Expr, MultiValue
+from pyteal.errors import TealTypeError, verifyTealVersion
+from pyteal.ir import Op, TealBlock, TealOp
+from pyteal.types import TealType, require_type
+
+if TYPE_CHECKING:
+    from pyteal.compiler import CompileOptions
+
+
+EcdsaPubkey = [TealType.bytes, TealType.bytes]
+
+
+class EcdsaCurve(Enum):
+    Secp256k1 = 0
```

Comment:
personal nit: maybe add a comment here (might sound redundant but could be something like: Enum representing the ECDSA curve used for the public key)